static/app/views/settings/components/settingsBreadcrumb/breadcrumbDropdown.spec.jsx

```diff
@@ -21,55 +19,39 @@ describe('Settings Breadcrumb Dropdown', () => {
     createWrapper();
     expect(screen.getByText('Test')).toBeInTheDocument();
     userEvent.hover(screen.getByText('Test'));
-    jest.runAllTimers();
     expect(screen.getByText('foo')).toBeInTheDocument();
     expect(screen.getByText('bar')).toBeInTheDocument();
   });
 
-  it('closes after 200ms when mouse leaves crumb', () => {
-    createWrapper();
-    userEvent.hover(screen.getByText('Test'));
-    jest.runAllTimers();
-    expect(screen.getByText('foo')).toBeInTheDocument();
-
-    userEvent.unhover(screen.getByText('Test'));
-    jest.advanceTimersByTime(10);
```

Comment:
Wasn't even testing it correctly since it didn't check that it was still in the document after unhover.